transforms/__testfixtures__/arrow-function.output.js

```diff
@@ -23,3 +23,39 @@ var fn6 = () => ({
 });
 
 [1, 2, 3].map(x => x * x);
+
+[1, 2, 3].map(x => x * x);
+
+compare(1, 2, (num1, num2) => num1 > num2);
+
+Promise.resolve()
+.then(function() {
+  console.log('foo');
+}.bind(this, 'a'))
+.then(a => 4);
+
+foo(function(a) {
+  this.bar(function() {
+    return a + this.b;
+  });
+});
+
+foo(function(a) {
```

Comment:
It appears that eslint's [prefer-arrow-callback](http://eslint.org/docs/rules/prefer-arrow-callback) rule wants this outer function to be an arrow function.